draft-gpew-priv-ppm.md

```diff
@@ -810,29 +810,42 @@ PPM task, the leader issues an "aggregate-share request" to each helper. The
 helper responds to this request by extracting its aggregate share from its state
 and encrypting it under the collector's HPKE public key.
 
-For each aggregator endpoint `[aggregator]` in the parameters associated with
-`CollectReq.task_id` (see {{pa-collect}}) except its own, the leader sends a
-POST request to `[aggregator]/aggregate_share` with the following message:
+First, the leader computes a checksum over the set of output shares included in
+the batch window. The checksum is computed by taking the SHA256 hash of each
+nonce from the client reports included in the aggregation, then combining the
+hash values with a bitwise-XOR operation.
```

Comment:
I don't object to this, really, but it seems like kind of overkill to use SHA-256.

Reply:
I agree, a non-cryptographic hash would be fine here IMO since we think we don't need a cryptographic hash, since the overall construction permits attackers to produce collisions anyway, and since using a cryptographic hash may give the false impression that the overall construction is actually secure.

I'm going to leave the OPEN ISSUE for now but we should consider addressing this -- either switching to a `checksum` algorithm that actually does provide collision-resistance over the set of nonces, or switch away from using cryptographic-strength primitives entirely.